.github/workflows/build.yml

```
@@ -7,47 +7,101 @@ on:
       - '*'
       - '!master'
 
+env:
+  IMAGE_NAME: ${{ github.repository_owner }}/node-base
+      
 jobs:
   build_matrix:
     strategy:
       matrix:
         # NOTE: These versions must be kept in sync with the release.yml
-        version: ["18.19.1", "20.11.1", "22.2.0"]
+        # In the case where a new node version introduces a breaking change,
+        # replace the major version of the matrix to a pinned version here.
+        # Ex: ["18", "20", "22"] ---> ["18.19.1", "20", "22"]
+        version: ["18.19.1", "20", "22"]
```

```suggestion
        version: ["18", "20", "22"]
```

Get rid of minor.patch.